modules/ROOT/pages/enabling-api-catalog.adoc

```diff
@@ -0,0 +1,33 @@
+= Enabling API Catalog
+
+Enable a connected Salesforce organization to access and sync Anypoint Platform APIs into the Salesforce API Catalog and expose them as invocable actions. This capability is enabled in Anypoint Platform Access Management by an Anypoint Platform organization administrator.  
```

Comment:
```suggestion
Enable a connected Salesforce organization to access and sync Anypoint Platform APIs into API Catalog in Salesforce. This capability is enabled in Anypoint Platform Access Management by an Anypoint Platform organization administrator.  
```

Reply:
Let's ask Ankita Darda (PM) about the level of detail we want to add. I'd suggest leaving off the details about what they can do once the APIs are in API Catalog and handle that in the API Catalog UI text and Help content. There's still some contention about what to call actions (invocable actions or just actions like the ASCX team does) and if we talk about those, we probably want to talk about topic and agent actions, which gets a little long. 

The naming is a little awkward, but it's not called Salesforce API Catalog right now. It's API Catalog in Salesforce. Again, this would be a PM question. Also, we shouldn't use "the API Catalog", since API Catalog is a proper noun.